src/main/java/de/unijena/cheminf/mortar/gui/views/SettingsView.java

```diff
@@ -206,15 +207,12 @@ private void addPropertyItems(GridPane aGridPane, List<Property<?>> aPropertiesL
             aRecentPropertiesMap.put(tmpPropName, tmpRecentValue);
             switch (tmpProperty) {
                 case SimpleBooleanProperty tmpSimpleBooleanProperty -> {
-                    ComboBox<Boolean> tmpBooleanComboBox = new ComboBox<>();
-                    tmpBooleanComboBox.setPrefWidth(GuiDefinitions.GUI_TEXT_FIELD_PREF_WIDTH_VALUE);
-                    tmpBooleanComboBox.setMaxWidth(GuiDefinitions.GUI_SETTINGS_TEXT_FIELD_MAX_WIDTH_VALUE);
-                    tmpBooleanComboBox.getItems().addAll(Boolean.FALSE, Boolean.TRUE);
-                    tmpBooleanComboBox.valueProperty().bindBidirectional(tmpSimpleBooleanProperty);
-                    tmpBooleanComboBox.setTooltip(tmpTooltip);
+                    ToggleSwitch tmpToggle = new ToggleSwitch();
+                    tmpToggle.setTooltip(tmpTooltip);
+                    tmpToggle.valueProperty().bindBidirectional(tmpSimpleBooleanProperty);
                     //add to gridpane
-                    aGridPane.add(tmpBooleanComboBox, 1, tmpRowIndex++);
-                    GridPane.setMargin(tmpBooleanComboBox, new Insets(GuiDefinitions.GUI_INSETS_VALUE));
+                    aGridPane.add(tmpToggle, 1, tmpRowIndex++);
+                    GridPane.setMargin(tmpToggle, new Insets(GuiDefinitions.GUI_INSETS_VALUE));
```

Comment:
When the setting dialog opens, I can see the switches turning on. Maybe set the initial state here already if it is on, before showing the dialog?